lib/ansible/modules/network/ios/ios_logging.py

```diff
@@ -47,7 +47,8 @@
   size:
     description:
       - Size of buffer. The acceptable value is in range from 4096 to
-        4294967295 bytes.
+        4294967295 bytes. If value of C(dest) is I(buffered), the default value
+        is 4096.
```

Comment:
you can have a key named `default` instead.
This should look like the following:
```
  size:
    description:
      - Size of buffer. The acceptable value is in range from 4096 to
        4294967295 bytes.
    default: 4096
```